src/routes/index.js

```diff
@@ -126,7 +135,18 @@ router.post(
           } else {
             user.emailVerified = true;
             user.markModified('emailVerified');
-            user.save();
+
+            const returnedSeaonNumber = await seaonNumber.findOne({}).exec();
+
+            const defaultRankData = new RankData(
+              {
+                username: req.body.username,
+                seasonNumber: returnedSeaonNumber.number,
+              }
+            );
```

Comment:
Do we need to do this for all users? Do users have to have a rating? Or can we just create one at the end of a game if it doesn't exist?